Projects/UOContent/Mobiles/Animals/Mounts/Ethereals.cs

```diff
@@ -212,68 +213,51 @@ public override void OnSingleClick(Mobile from)
         {
             base.OnSingleClick(from);
 
-            LabelTo(from, m_IsDonationItem ? "Donation Ethereal" : "Veteran Reward");
+            LabelTo(from, _isDonationItem ? "Donation Ethereal" : "Veteran Reward");
         }
 
-        public override void Serialize(IGenericWriter writer)
+        private void Deserialize(IGenericReader reader, int version)
         {
-            base.Serialize(writer);
-
-            writer.Write(3); // version
-
-            writer.Write(m_IsDonationItem);
-            writer.Write(IsRewardItem);
-
-            writer.Write(m_MountedID);
-            writer.Write(m_RegularID);
-            writer.Write(m_Rider);
-        }
-
-        public override void Deserialize(IGenericReader reader)
-        {
-            base.Deserialize(reader);
-            LootType = LootType.Blessed;
-
-            var version = reader.ReadInt();
-
             switch (version)
             {
                 case 3:
                     {
-                        m_IsDonationItem = reader.ReadBool();
+                        _isDonationItem = reader.ReadBool();
                         goto case 2;
                     }
                 case 2:
                     {
-                        IsRewardItem = reader.ReadBool();
+                        _isRewardItem = reader.ReadBool();
                         goto case 0;
                     }
                 case 1:
                     reader.ReadInt();
                     goto case 0;
                 case 0:
                     {
-                        m_MountedID = reader.ReadInt();
-                        m_RegularID = reader.ReadInt();
-                        m_Rider = reader.ReadEntity<Mobile>();
-
-                        if (m_MountedID == 0x3EA2)
+                        _mountedID = reader.ReadInt();
+                        _regularID = reader.ReadInt();
+                        _rider = reader.ReadEntity<Mobile>();
+                        if (_mountedID == 0x3EA2)
                         {
-                            m_MountedID = 0x3EAA;
+                            _mountedID = 0x3EAA;
                         }
-
                         break;
                     }
             }
-
             AddFollowers();
-
             if (version < 3 && Weight == 0)
```

Comment:
We can get rid of this weight fix.

Reply:
can i remove AddFollowers(); insidemethod? because it is alredy inside AfterDeserialization()